tedge/src/command.rs

```diff
@@ -178,37 +179,11 @@ pub struct BuildContext {
 /// ```
 pub struct ExecutionContext {
     pub user_manager: UserManager,
+    pub system_service_manager_factory: Box<dyn SystemServiceManagerFactory>,
 }
 
 impl ExecutionContext {
-    /// Build a new execution context.
-    ///
-    /// Such a context MUST be created only once,
-    /// in practice in the `main()` function.
-    pub fn new() -> ExecutionContext {
-        ExecutionContext {
-            user_manager: UserManager::new(),
-        }
-    }
```

Comment:
Why remove the constructor? It seems convenient to have one, especially that after this change one has to provide a factory.

Reply:
I guess, because I got rid of `ExecutionContext` completely...